demo/src/main/java/com/orange/ods/demo/ui/components/chips/SubComponentChipFilter.kt

```diff
@@ -0,0 +1,102 @@
+import com.orange.ods.demo.ui.utilities.composable.Subtitle
+
+@ExperimentalMaterialApi
+@Composable
+fun SubComponentChipFilter() {
+    val customizationState = rememberChipCustomizationState()
+
+    ComponentCustomizationBottomSheetScaffold(
+        bottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
+        bottomSheetContent = {
+
+            Subtitle(textRes = R.string.component_element_leading, withHorizontalPadding = true)
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = dimensionResource(id = R.dimen.ods_spacing_xs))
+                    .padding(start = dimensionResource(id = R.dimen.ods_spacing_s)),
+                horizontalArrangement = Arrangement.Start
+            ) {
+                LabelledRadioButton(
+                    selectedRadio = customizationState.leadingElement,
+                    currentRadio = ChipCustomizationState.LeadingElement.None,
+                    label = stringResource(id = R.string.component_element_none)
+                )
+                LabelledRadioButton(
+                    selectedRadio = customizationState.leadingElement,
+                    currentRadio = ChipCustomizationState.LeadingElement.Avatar,
+                    label = stringResource(id = R.string.component_element_avatar)
+                )
+            }
+
+            ComponentCustomizationCheckboxItem(labelRes = R.string.component_state_outlined, checked = customizationState.outlinedChecked)
+            ComponentCustomizationCheckboxItem(labelRes = R.string.component_state_disabled, checked = customizationState.disabledChecked)
+
+        }) {
+        Column(
+            modifier = Modifier
+                .verticalScroll(rememberScrollState())
+                .padding(
+                    horizontal = dimensionResource(id = R.dimen.ods_screen_horizontal_margin),
+                    vertical = dimensionResource(id = R.dimen.ods_screen_vertical_margin)
+                )
+        ) {
+            FlowRow(modifier = Modifier.fillMaxWidth()) {
+                for (index in 1..4) {
+                    FilterChip(index, customizationState)
+                }
+            }
+        }
+    }
+}
+
+@ExperimentalMaterialApi
+@Composable
+private fun FilterChip(index: Int, customizationState: ChipCustomizationState) {
+    val selected = rememberSaveable { mutableStateOf(false) }
+    OdsFilterChip(
+        text = "${stringResource(id = R.string.component_chip_filter)} $index",
+        leadingAvatar = if (customizationState.hasLeadingAvatar) painterResource(id = R.drawable.placeholder_small) else null,
+        onClick = { selected.value = !selected.value },
+        selected = selected.value,
+        outlined = customizationState.outlinedChecked.value,
+        enabled = !customizationState.disabledChecked.value,
+    )
+    Spacer(modifier = Modifier.size(dimensionResource(id = R.dimen.ods_spacing_xs)))
```

Comment:
We should use `mainAxisSpacing` parameter of `FlowRow` instead of adding a spacer here.

```kotlin
FlowRow(modifier = Modifier.fillMaxWidth(), mainAxisSpacing = dimensionResource(id = R.dimen.ods_spacing_xs))
```

Otherwise a line can begin with a space:

![Screenshot_20220629_163653](https://user-images.githubusercontent.com/13621389/176464567-81d0bc74-311a-4597-a167-e6f0464b4f60.png)